src/substrate/filters/labeler.ts

```diff
@@ -90,21 +90,21 @@ export const Label: LabelerFilter = (
       const { authorityId } = data;
       return {
         heading: 'Heartbeat Received',
-        label: ` A new heartbeat was received from ${fmtAddr(authorityId)}.`,
+        label: `A new heartbeat was received from ${fmtAddr(authorityId)}.`,
       };
     }
     case EventKind.SomeOffline: {
       const { sessionIndex } = data;
       return {
         heading: 'Some Offline',
-        label: `At the end of the session: ${sessionIndex}, at least one validator was found to be offline.`,
+        label: `At the end of session ${sessionIndex}, at least one validator was found to be offline.`,
       };
     }
     case EventKind.AllGood: {
       const { sessionIndex } = data;
       return {
         heading: 'All Good',
-        label: `At the end of the session: ${sessionIndex}, no offence was committed.`,
+        label: `At the end of session ${sessionIndex}, no offence was committed.`,
```

Comment:
Maybe reword to "No offences were committed at the end of session X". Again, though, we may want to avoid passive voice and say "No validators committed offences during session X".